src/colorizer/Collection.ts

```diff
@@ -331,7 +331,7 @@ export default class Collection {
    * @returns a new Collection, where the only dataset is that of the provided `datasetUrl`.
    * The `url` field of the Collection will also be set to `null`.
    */
-  public static makeCollectionFromSingleDataset(datasetUrl: string): Collection {
+  public static async makeCollectionFromSingleDataset(datasetUrl: string): Promise<Collection> {
```

Comment:
Q: I see a couple new `await`s to support this `async` (farther down this file, in Viewer.tsx, and in tests), but no new `await`s within this function to justify the `async`. Is this in anticipation of some async behavior you plan to add later?

Reply:
Ah, removed this. I thought I'd reversed this change, but I may have lost it while I was squashing some commits on the previous branch.